test/object-store/sync/flx_migration.cpp

```diff
@@ -230,7 +230,64 @@ TEST_CASE("Test server migration and rollback", "[flx],[migration]") {
 
 #ifdef REALM_SYNC_PROTOCOL_V8
 
-TEST_CASE("Validate protocol v8 features", "[flx],[migration]") {
+TEST_CASE("Test client migration and rollback", "[flx][migration]") {
```

Comment:
I agree we need those tests. I have a list with a bunch of tests I want to add, maybe I'll create some of them for this PR, otherwise we have tickets for tests.